src/platform/forms-system/src/js/web-component-patterns/relationshipToVeteranPattern.jsx

```diff
@@ -0,0 +1,57 @@
+import { radioUI, radioSchema } from './radioPattern';
+import VaTextInputField from '../web-component-fields/VaTextInputField';
+
+export const relationshipToVeteranUI = personTitle => {
+  const person = personTitle ?? 'veteran';
+
+  return {
+    relationshipToVeteran: radioUI({
```

Comment:
I realized after the fact that even the name of this field assumes 'veteran'. Just something to keep in mind later.